src/SDK/SmartDetectors.ProjectTemplate/EmptyAlert.cs

```diff
@@ -1,6 +1,7 @@
 ﻿namespace $safeprojectname$
 {
     using Microsoft.Azure.Monitoring.SmartDetectors;
+    using Microsoft.Azure.Monitoring.SmartDetectors.AlertPresentation;
```

Comment:
not necessary

Reply:
i'd rather leave it, to save the developer the import

---
In reply to: [280450306](https://github.com/Azure/SmartDetectorsMonitoringAppliance/pull/62#discussion_r280450306) [](ancestors = 280450306)